_deps/polkadot/api.ts

```diff
@@ -0,0 +1 @@
+export * from "https://deno.land/x/polkadot@0.0.4/api/mod.ts";
```

Comment:
This version of `polkadot` is maintained by you @harrysolovay ?

Reply:
No, it's maintained by Jaco.